subrpcserver_config.go

```diff
@@ -243,13 +244,16 @@ func (s *subRPCServerConfigs) PopulateDependencies(cfg *Config,
 		case *wtclientrpc.Config:
 			subCfgValue := extractReflectValue(subCfg)
 
-			if towerClient != nil {
+			if towerClient != nil && anchorTowerClient != nil {
```

Comment:
why `&&`? Because none or both are active?

Reply:
yeah, currently it's only possible to activate both. we could do them separately if we think that's better